CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if( ${TARGET_OS} STREQUAL "Linux" )
 elseif(${TARGET_OS} STREQUAL "Windows" )
   set(OS "WINDOWS")
   set(OSNAME "Windows")
-  add_definitions(-DWIN32 -DHOST_WIN -DWINNT -DTARGET_WIN -DTARGET_WIN_X86)
+  add_definitions(-DWIN32 -DHOST_WIN -DWINNT -DTARGET_WIN -DTARGET_WIN_X86 -DPGDLL)
```

Comment:
This should only be defined for `_shared` targets only.

Reply:
`flangmain` is compiled only once. Should we just add this to flangmain specifically?

Reply:
See here: https://github.com/flang-compiler/flang/blob/a9ccdd766d5e606a1dc073fa3b106e52bab5dc64/runtime/flangmain/flangmain.c#L67

Reply:
flangmain is compiled as a static library, so it's not needed there.